aspnetcore/grpc/migration.md

```diff
@@ -81,20 +81,17 @@ ASP.NET Core [middleware](xref:fundamentals/middleware/index) offers similar fun
 * Are used to construct a pipeline that handles a gRPC request.
 * Allow work to be performed before or after the next component in the pipeline.
 
-gRPC Interceptors differences from ASP.NET Core Middleware:
+gRPC Interceptor differences from ASP.NET Core Middleware:
 
-* Middleware operates on the underlying HTTP/2 messages.
 * Interceptors operate on the gRPC layer of abstraction using the [ServerCallContext](https://grpc.io/grpc/csharp/api/Grpc.Core.ServerCallContext.html).
 * ASP.NET Core middleware runs before gRPC interceptors.
+* Middleware operates on the underlying HTTP/2 messages.
 * ASP.NET Core middleware provides access to `HttpContext`, `HttpRequest`, and `HttpResponse`.
 * gRPC interceptors provide access to:
```

Comment:
Add a bullet point:

* `HttpContext` with `ServerCallContext.GetHttpContext()` extension method.

Reply:
Since both middleware and interceptors have access to HttpContext, move this and "ASP.NET Core middleware provides access to `HttpContext`, `HttpRequest`, and `HttpResponse`." to the both section:

* Provide access to `HttpContext`. The `HttpContext` is an parameter passed to middleware. Inside an interceptors you can access the `HttpContext` using the `ServerCallContext.GetHttpContext()`